.github/workflows/release_linux_x86_64.yml

```diff
@@ -47,6 +47,14 @@ jobs:
         python -m pip install pytest==5.4.3 nbval
         pytest
 
+    - name: Test backend test data
+      run: |
+        # onnx.checker all existing backend data
+        python workflow_scripts/test_generated_backend.py
+        # onnx.checker all generated backend data
+        python onnx/backend/test/cmd_tools.py generate-data
+        python workflow_scripts/test_generated_backend.py
+
```

Comment:
TODO: This duplicate code among all release CIs will be merged into a same yaml and be reused in the future.
https://github.com/actions/starter-workflows/issues/245 Currently GitHub Action has not supported reused yaml yet.